homeassistant/components/ps4/media_player.py

```diff
@@ -287,8 +280,9 @@ def reset_title(self):
             self._media_type = media_type
 
             self.update_list()
-            self.schedule_update()
+            self.async_write_ha_state()
 
+    @callback
```

Comment:
`update_list` is not async safe. We're loading games which loads a json file.

Reply:
Good catch, fixed.